### DISCUSSION TITLE
A note about my editied posts

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1212] [Team Posts]
It occurred to me that you are reading my posts via email rather than here 
 on the GitHub site ? I can tell by your quotes. Thus you get my first drafts, and no edits ?
Forgive me if you receive drafts where I say something crazy.
That's why we have edits. Please come here to read them.

**Comment by kybos** [comment]
Yes, the notification mails are sent automatically, probably each time you press "Comment".
No big issue for me, as I always come here to github when writing an answer or looking into something in more detail.
